libs/sdk-ui-dashboard/.dependency-cruiser.js

```diff
@@ -8,7 +8,11 @@ options = {
 
         depCruiser.isolatedSubmodule("constants", "src/presentation/constants"),
         depCruiser.isolatedSubmodule("localization", "src/presentation/localization"),
-        depCruiser.isolatedSubmodule("presentationComponents", "src/presentation/presentationComponents"),
+        depCruiser.moduleWithDependencies(
+            "presentationComponents",
+            "src/presentation/presentationComponents",
+            ["src/model", "src/model/store/ui/uiSelectors.ts"],
```

Comment:
Should the src/model export the necessary selectors? So that this module can import it from src/model?